packages/inference/src/types.ts

```diff
@@ -95,6 +96,7 @@ export interface ProviderConfig {
 	makeBody: (params: BodyParams) => Record<string, unknown>;
 	makeHeaders: (params: HeaderParams) => Record<string, string>;
 	makeUrl: (params: UrlParams) => string;
+	closedSource?: boolean;
```

Comment:
```suggestion
	clientSideRoutingOnly?: boolean;
```
suggestion of prop name